common/changes/@gooddata/sdk-ui-all/master_2022-01-26-15-36.json

```diff
@@ -0,0 +1,10 @@
+{
+    "changes": [
+        {
+            "packageName": "@gooddata/sdk-ui-all",
+            "comment": "Implementation of ScrollGradient with possibilities to define custom background color and size of gradient overlay",
```

Comment:
We do not mention the changes to sdk-ui-kit in the changelog, plese remove this file (thanks for running the rush change though :) )

Reply:
Done! :)